lighthouse-core/scripts/i18n/prune-obsolete-lhl-messages.js

```diff
@@ -135,7 +135,7 @@ function pruneObsoleteLhlMessages() {
   const alreadyLoggedPrunes = new Set();
   for (const localePath of localePaths) {
     const absoluteLocalePath = path.join(lhRoot, localePath);
-    const localeLhl = require(absoluteLocalePath);
+    const localeLhl = JSON.parse(fs.readFileSync(absoluteLocalePath, 'utf-8'));
```

Comment:
> it certainly wasn't obvious to me debugging that it's run in the same process as collect-strings :)

this may be one of those things that's obvious while writing it but isn't when running it, so we could change it. The idea was that `collect-strings` collects then creates an updated `en-US.json`, so it made sense at that point to make sure none of the existing LHL files conflicted with the new `en-US.json`.

> and the error manifests in the update command being a noop because we use the pre-overriden strings instead of the new ones

what action is being done that's a no-op? #9598 definitely removed strings after running `collect-strings.js`. What else in there is loading the locale json files and overwriting them with the cached version?

Reply:
> what action is being done that's a no-op? 

the entire script command that updates the locales from tc becomes a no-op because this function just overwrites them with the locales that were there originally (i.e. it "prunes" the cached copy loaded `require` which is old)

Reply:
> the entire script command that updates the locales from tc becomes a no-op because this function just overwrites them with the locales that were there originally

ah, see @exterkamp, this is why that script should be in this repo :P